src/main/kotlin/deck/Deck.kt

```diff
@@ -0,0 +1,21 @@
+package deck
+
+import card.CardObject
+
+class Deck(inpCards: MutableList<CardObject>) {
+    /*
+    Deck contains a list of cards. Deck class is a wrapper for operations on that list
+     */
+    var cards: MutableList<CardObject> = inpCards
+
+    //Use internal list shuffling
+    fun shuffle(){cards.shuffled()}
+
+
+    //Remove some card from the deck with matching name? id? something we need to figure out
+    //Probably a cleaner way to do this - but couldn't find one
+    fun remove(name: String){
+        cards.remove(cards.filter{ x -> x.name == (name)}[0])
```

Comment:
```suggestion
        cards.remove(cards.filter{ x -> x == card}[0])
```

This probably also needs some error checking, but we can worry about that a bit later.